Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -191,10 +193,13 @@ bool VolumeVerifier::CheckPartition(const Partition& partition)
                          name.c_str()));
   }
 
-  CheckCorrectlySigned(
-      partition,
-      StringFromFormat(Common::GetStringT("The %s partition is not correctly signed.").c_str(),
-                       name.c_str()));
+  if (!m_is_datel)
+  {
+    CheckCorrectlySigned(
+        partition,
+        StringFromFormat(Common::GetStringT("The %s partition is not correctly signed.").c_str(),
+                         name.c_str()));
```

Comment:
Do you think it makes sense to also migrate this to fmt?

Reply:
I'd say it's better to save that for a PR that migrates all of VolumeVerifier's `StringFromFormat` calls at once.